src/index.ts

```diff
@@ -66,18 +64,23 @@ class LocalizeAssetsPlugin<LocalizedData = string> {
 		this.localizeCompiler = (
 			this.options.localizeCompiler
 				? this.options.localizeCompiler
-				: function (localizerArguments) {
-					const [key] = localizerArguments;
-
-					if (localizerArguments.length > 1) {
-						this.emitWarning(`[${name}] Ignoring confusing usage of localization function: ${functionName}(${localizerArguments.join(', ')})`);
-						return key;
-					}
+				: {
+					[this.options.functionName ?? '__'](localizerArguments) {
+						const [key] = localizerArguments;
+
+						if (localizerArguments.length > 1) {
+							const code = stringifyAst(this.callNode);
+							this.emitWarning(`[${name}] Ignoring confusing usage of localization function: ${code})`);
+							return key;
+						}
 
-					const keyResolved = this.resolveKey();
-					return keyResolved ? JSON.stringify(keyResolved) : key;
+						const keyResolved = this.resolveKey();
+						return keyResolved ? JSON.stringify(keyResolved) : key;
+					},
```

Comment:
Do you think there are any cases where they would want this default localizer in `__`, and just add on extra localizers?

I might be overthinking this but was wondering if it's worth exporting this default one if there's a use-case for it.

Reply:
Yeah, I can see an argument for that. You might also want to delegate to the default one - like, try to do some custom logic but fall back to the default one if it failed, that sort of thing.

The default compiler is not exactly private anyway, you can't easily change its behaviour and its exact implementation is documented in the readme, so exposing it doesn't really cost anything and saves clients some copy+pasting.

A pedestrian question: how do you envision exposing the default compiler? `import LocalizeAssetsPlugin, { defaultLocalizeCompiler } from "webpack-localize-assets-plugin"` would work but it's a bit clunky (... in fact I think it might not even work correctly in a commonjs context?)

We could alternatively expose the default as a static prop on the class itself, so you'd `import LocalizeAssetsPlugin from "webpack-localize-assets-plugin"` and then access it as `LocalizeAssetsPlugin.defaultLocalizeCompiler` or something. What do you think?

Reply:
Could even make it an instance property on the plugin, although that feels extremely clunky to me:

```javascript
var localizeCompiler = {
    __() {
        // ...
        pluginInstance.defaultLocalizeCompiler.call(this, ...arguments);
    }
};
var pluginInstance = new WebpackLocalizeAssetsPlugin({ localizeCompiler });
module.exports = {
    // ...
    plugins: [plugin]
};
```

Reply:
I think static member on the class is the only good option TBH.

Reply:
00433f4. LMK what you think!

Reply:
Looks great! Thank you